.github/workflows/tutorials.yml

```diff
@@ -0,0 +1,105 @@
+name: Jupyter Notebooks
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  tutorials:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+
+    env:
+      DEVITO_ARCH: "${{ matrix.compiler }}"
+      DEVITO_OPENMP: ${{ matrix.openmp }}
+      DEVITO_BACKEND: "core"
+      PYTHON_VERSION: "3.7"
+      RUN_CMD: ""
+
+    strategy:
+      # Prevent all build to stop if a single one fails
+      fail-fast: false
+      matrix:
+        name: [
+          ubuntu-16.04-gcc-py37,
+          macOS-10.14-gcc-py37,
+          macOS-10.14-clang-py37,
+          docker-devito
+          ]
+
+        include:
+          - name: ubuntu-16.04-gcc-py37
+            os: ubuntu-16.04
+            compiler: gcc-7
+            openmp: 1
+
+          - name: macOS-10.14-gcc-py37
+            os: macos-latest
+            compiler: gcc-9
+            openmp: 1
+
+          - name: macOS-10.14-clang-py37
+            os: macos-latest
+            compiler: clang
+            openmp: 0
+
+          - name: docker-devito
+            os: ubuntu-latest
+            compiler: gcc
+            openmp: 1
+
+    steps:
+    - name: Checkout devito
+      uses: actions/checkout@v1
+
+    - name: Set up Python 3.7
+      if: matrix.name != 'docker-devito'
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+
+    - name: Install compilers for OSX
+      if: runner.os == 'macOS'
+      run: |
+        if [ "${{ matrix.compiler }}" = "gcc-9" ]; then
+          brew install gcc
+        else
+          sudo xcode-select -s /Applications/Xcode_11.app/Contents/Developer
+        fi
+        # dask error on osx, skip dask tuto
+        echo "::set-env name=SKIP::--deselect=examples/seismic/tutorials/04_dask.ipynb"
```

Comment:
I think this should be in the test file, not in the action. 

Could we mark it (pseudocode) `xfail(condition=OS is MacOS, reason=sympy+dask not behaving)`

I'd also say file an issue for this MacOS + sympy + dask bug, plus a reproducer

Reply:
That's a slow tutorial to run bit overkill to run it with xfail, can only mark cells not the file with nbval

Reply:
So we are conflating two issues here:

- slowness
- macos + dask + sympy bug (which btw needs a reproducer)

Let's ignore slowness for a moment. Can we xfail individual cells? I doubt we can (based on what you just said). But if we could, I'd by far prefer the xfail. Otherwise, we'll have to leave it the way it is now.

Reply:
Can xfail a single cell not sure how nbval handles conditional xfail, gotta look if feasible

Reply:
So far, all iI found is `NBVAL_SKIP` but it's just a flag that doesn't take input so would skip it for all builds. Still digging but skipping the notebook for osx probably the simplest. The whole notebook fails anyway so skipping it isn't really an issue, but yes need to be said somewhere for users that there is an issue. I have no idea where to file that issue though

Reply:
I think we need an MFE?

Remind me -- you cannot reproduce the behaviour on your mac right ?

Reply:
I can, fails too on my mac with same error, MFE sure don't really have time to look into it right now but will try.